internal/internal_test.go

```diff
@@ -88,7 +89,7 @@ func TestCombinedOutput(t *testing.T) {
 	out, err := CombinedOutputTimeout(cmd, time.Second)
 
 	assert.NoError(t, err)
-	assert.Equal(t, "foo\n", string(out))
+	assert.Equal(t, "foo"+testutil.LineSeparator, string(out))
```

Comment:
I've mixed feelings on if these tests should just be deleted, but I see testing it proper would be a fair bit of work.  We could instead run a command using the `GO_WANT_HELPER_PROCESS` method and then we can determine the line separator.